grid_maker.py

```diff
@@ -0,0 +1,51 @@
+
+import sys
+
+class Grid_Maker:
+    """
+    Create a user defined grid with error check
+    Returns a formatted string
+    Parameters are:
+    - s (side character of grid)
+    - t (top character of grid)
+    - c (corner character of grid)
+    - h (number of side characters)
+    - w (number of top characters)
+    - b (number of boxes)
+    """
+    # Initialize
+    def __init__(self, s = '|', t = '-', c = '+', h = 5, w = 5, b = 2):
+        self.s = s
+        self.t = t
+        self.c = c
+        self.h = h
+        self.w = w
+        self.b = b
```

Comment:
You should rename the function names what you have in the docstring here
`self.side_character_of_grid` = s` and so forth